apps/patient/src/api/external.ts

```diff
@@ -16,7 +16,7 @@ export const geocode = async (address: string) => {
     } else {
       throw new Error('No results found for the provided address.');
     }
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
i thiiiiiink it's `google.maps.GeocoderError`, but not sure

Reply:
If the line on line `17` fires, it'll just be `Error`

Reply:
ah 👍🏼